ui/design-system-components/src/components/Entry/EntryCard/index.tsx

```diff
@@ -148,7 +148,7 @@ const EntryCard: React.FC<EntryCardProps> = props => {
   const contentClickableStyle =
     contentClickable && !showNSFWCard ? 'cursor-pointer' : 'cursor-default';
   const menuItems: ListItem[] = [
-    ...(!isViewer
+    ...(!isViewer && flagAsLabel
```

Comment:
What if we don't want the label but we still want to see the menu with just the icon ? We can do
`label: flagAsLabel ?? ''`

Reply:
That could work too. Although we do not have any such scenario yet

Reply:
It doesn't make sense to remove a menu just because you don't have a label. If at some point we want icon only menu item the logic you have fails. Rather it would make sense if you check for `onEntryFlag`. If that prop doesn't exist it definitely means we don't a flagging logic but having empty label doesn't mean the same.